utils/create-jobs.py

```diff
@@ -46,7 +46,7 @@ def main():
             args.job_name += " - {}".format(args.job_index)
 
     job = ajt.render_job(args.urlbase, args.machine, tests = args.tests, priority = args.priority,
-                         rfs_type = args.rfs_type[0], job_name = args.job_name)
+                         rfs_type = args.rfs_type, job_name = args.job_name)
```

Comment:
so you're passing a list now ? Funny, why did it change ?

Reply:
rfs_type is no longer a list